src/test/java/de/tum/in/www1/artemis/ArchitectureTest.java

```diff
@@ -58,4 +60,10 @@ void testNoJunitJupiterAssertions() {
 
         noJunitJupiterAssertions.check(testClasses);
     }
+
+    @Test
+    void testNoCollectorsToList() {
+        ArchRule foo = noClasses().should().callMethod(Collectors.class, "toList").because("You should use .toList() or .collect(Collectors.toCollection(ArrayList::new)) instead");
+        foo.check(allClasses);
```

Comment:
```suggestion
        ArchRule toListUsage = noClasses().should().callMethod(Collectors.class, "toList").because("You should use .toList() or .collect(Collectors.toCollection(ArrayList::new)) instead");
        toListUsage.check(allClasses);
```